test/unit/ExecuteQueryHandler-test.js

```diff
@@ -9,21 +9,21 @@ describe('a ExecuteQueryHandler instance', () => {
       const path = { settings: {}, toString: () => 'path' };
       const iterable = handler.execute(path, {});
       const iterator = iterable[Symbol.asyncIterator]();
-      await expect(iterator.next()).rejects
+      expect(iterator.next()).rejects
```

Comment:
`async` is needed here: `rejects` returns a promise

Reply:
You're right. I usually `return` those `expect`'s in async tests, which is probably part of my confusion. I'll amend the last commit.